src/ensembl/src/shared/components/loader/Loader.scss

```diff
@@ -15,3 +15,44 @@
     transform: rotate(360deg);
   }
 }
+
+.listLoader {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  section {
+    margin-top: 45px;
+  }
+
+  div {
+    background-color: $medium-light-grey;
+    border-radius: 20px;
+    height: 9px;
+    margin-bottom: 10px;
+    animation: blinking-background 1.3s linear infinite;
+  }
+
+  .listHeading {
+    width: 50%;
+  }
+
+  .shortItem {
+    width: 70%;
+  }
+
+  .longItem {
+    width: 100%;
+  }
+}
+
+@keyframes blinking-background {
+  0%, 50%, 100% {
+    background-color: $light-grey;
+  }
+
+  25%, 75% {
+    background-color: $medium-light-grey;
+  }
+}
```

Comment:
just in case we want the moving gradient rather than the blinking (I asked Andrea for opinion), here's a code snippet:

```css

  div {
    background-color: $medium-light-grey;
    border-radius: 20px;
    height: 9px;
    margin-bottom: 10px;
    animation: placeholder-shimmer 8s linear infinite;
    background-image: linear-gradient(to right, #e4e9ee 4%, #{$medium-light-grey} 25%, #e4e9ee 36%);
  }


...


@keyframes placeholder-shimmer {
  0% {
    background-position: -1000px 0
  }
  100% {
    background-position: 1000px 0
  }
}
```

result: 


https://user-images.githubusercontent.com/6834224/132779814-0a5fb988-5c2b-44e2-936a-9fb503788c01.mp4